test/app/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "devDependencies": {
     "ts-loader": "6.2.1",
-    "typescript": "3.0.1",
```

Comment:
Stable 3.8.0 and 3.8.1 don't exist: they were used as dev and rc versions. Let's use the first 3.8 stable version, which is 3.8.2